05_book_titles/book.rb

```diff
@@ -1,3 +1,21 @@
 class Book
-# write your code here
+    attr_accessor :title
```

Comment:
Because you immediately override the writer method below (`def title=`), there's no need to use `attr_accessor`, which is `attr_writer` and `attr_reader` combined.

```suggestion
    attr_reader :title
```